spikingjelly/activation_based/op_counter/spikesim/formulas.py

```
@@ -0,0 +1,78 @@
+    if active_row_count_by_tile is not None and input_tile_channels is not None:
+        if len(active_row_count_by_tile) != len(input_tile_channels):
+            raise ValueError(
+                "active_row_count_by_tile and input_tile_channels must have the "
+                f"same length, but got len(active_row_count_by_tile)="
+                f"{len(active_row_count_by_tile)} and len(input_tile_channels)="
+                f"{len(input_tile_channels)}."
+            )
+        xbar_rows = 0.0
+        for row_count, tile_channels in zip(
+            active_row_count_by_tile, input_tile_channels
+        ):
```

Ruff (0.15.12)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)